### PR TITLE
[DOCS] Add missing comma in session example

### PR DESCRIPTION
**Pull request by AngelOnFira**
When I copied this code to my editor, the linter expected a comma here.

### CODE REVIEW

File: content/api/commands/session.md [hunk 1]
```diff
@@ -339,19 +339,19 @@ const login = (name = 'user1') => {
...
 it('can view profile', () => {
-  cy.login()
+  login()
 })
 
 // add_blog.cy.js
 it('can create a blog post', () => {
-  cy.login()
+  login()
 })
```
Comment (AngelOnFira) [inline content/api/commands/session.md:355]: It also seems that `cy.` needs to be removed here to reference the function above.

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@AngelOnFira thank you!!

**Comment by AngelOnFira** [comment]
It looks like more docs on this page are also a bit inconsistent. I'm also running into some problems that are described in https://github.com/cypress-io/cypress/issues/25272, so I might take a crack at expanding the information here a bit more before this is merged.

I could also do it in a separate PR if that would be better :)

**Review by elylucas — APPROVED** [review]

**Comment by elylucas** [comment]
thanks @AngelOnFira

**Comment by elylucas** [comment]
> It looks like more docs on this page are also a bit inconsistent. I'm also running into some problems that are described in [cypress-io/cypress#25272](https://github.com/cypress-io/cypress/issues/25272), so I might take a crack at expanding the information here a bit more before this is merged.
> 
> I could also do it in a separate PR if that would be better :)

Yes, new PR please, thanks!

**Pull request merged**